internal/git/cli.go

```diff
@@ -424,6 +456,20 @@ func (g *GitCLI) RemoteAdd(name string, url string) error {
 	return g.run("remote", "add", name, url)
 }
 
+func (g *GitCLI) Merge(repositoryURL, baseSHA, headSHA, committerDate, workingDir string) (string, error) {
+	if g.mergeBinary == "" {
+		return "", fmt.Errorf("merge binary not found")
+	}
+
+	stdout, err := g.runMerge("merge", "--clone-url", repositoryURL,
+		"--base-sha", baseSHA,
+		"--head-sha", headSHA,
+		"--committer-date", committerDate,
+		"--working-dir", workingDir)
```

Comment:
Why specify the working directory again here when it was already specified in line 119?

Reply:
Done - updated.